css/index.css

```diff
@@ -23,13 +23,70 @@ time, mark, audio, video {
 	font: inherit;
 	vertical-align: baseline;
 }
+
+
+/* Add a black background color to the top navigation */
+.topnav {
+    background-color: #ffffff;
+    overflow: hidden;
+    text-align: center; 
+    margin-left: 400px; 
+    padding-left: 50px; 
+    padding-right: 50px;
+  }
+
+  .topnav a {
+    float: left;
+    padding: 30px;
+    color: white;
+    text-decoration: none;
+    font-size: 17px;
+    width: 14%; /* Four equal-width links. If you have two links, use 50%, and 33.33% for three links, etc.. */
+    text-align: center; /* If you want the text to be centered */
+  }
+  
+  /* Style the links inside the navigation bar */
+  .topnav a {
+    display: flex;
+    flex-direction: wrap;
+    float: left;
+    color: #000000;
+    text-align: right/*center*/;
+    padding: 14px 16px;
+    text-decoration: none;
+    font-size: 17px;
+  }
+
+.inner-section {
+    display: flex; 
+    /*flex: 1 0 25%; */
+    flex-direction: row;  
+    flex-wrap: wrap; 
+}
+
+body .container-about-page .inner-section {
+    display: flex; 
+    flex-wrap: wrap;
+    width: 50%;
+    margin-left: 27.5%; 
+}
+
+#About {
+    align-items: center;
+    justify-content: center;
+    /*margin-right: -50%; */
```

Comment:
Awesome, I love how you place comments above some of the classes like in like 48. That being said, in lines 62,77,87,88, etc... you have commented out code. If you don't need it, just delete it :) Keeping your codebase clean of unused code is a great habit to build early on.